server/delphilint-server/src/main/java/au/com/integradev/delphilint/analysis/EngineStartupConfiguration.java

```diff
@@ -38,7 +38,7 @@ public Set<Path> getPluginPaths() {
 
   public Map<String, String> getBaseProperties() {
     return Map.of(
-        "sonar.delphi.bds.path", bdsPath,
+        "sonar.delphi.installationPath", bdsPath,
         "sonar.delphi.compiler.version", compilerVersion);

```

Comment:
The `sonar.delphi.compiler.version` property was also renamed to `sonar.delphi.compilerVersion` in SonarDelphi v1.0.0.